### ISSUE TITLE
Startup Runtime Exception - Null Reference

### ISSUE THREAD

**Issue by Arion-Kun** [Arion-Kun/StartupManager#3]
Sometimes on Startup the program throws a null ref from AddRange()

This could be caused due to race conditions with threads as some async is involved yielding to the main thread.

A solution off the top of my head would be just to use a single thread to allocate the entries, which might cause some visual lag.

**Comment by Arion-Kun** [comment]
Exception dump for bugfix

```
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.InvalidOperationException: At least one of the provided rows is null.
   at System.Windows.Forms.DataGridView.OnAddingRows(DataGridViewRow[] dataGridViewRows, Boolean checkFrozenStates)
   at System.Windows.Forms.DataGridViewRowCollection.AddRange(DataGridViewRow[] dataGridViewRows)
   at StartupManager.Pages.StartupForm.<GetDataAsync>d__11.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()


************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4510.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/mscorlib.dll
----------------------------------------
StartupManager
    Assembly Version: 1.0.1.3
    Win32 Version: 1.0.1.3
    CodeBase: file:///C:/Program%20Files%20(x86)/Startup%20Manager/StartupManager.exe
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4488.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4488.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4395.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4341.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4470.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4161.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
System.Resources.Extensions
    Assembly Version: 4.0.1.0
    Win32 Version: 4.700.20.21406
    CodeBase: file:///C:/Program%20Files%20(x86)/Startup%20Manager/System.Resources.Extensions.DLL
----------------------------------------
System.Memory
    Assembly Version: 4.0.1.1
    Win32 Version: 4.6.28619.01
    CodeBase: file:///C:/Program%20Files%20(x86)/Startup%20Manager/System.Memory.DLL
----------------------------------------
System.Runtime.CompilerServices.Unsafe
    Assembly Version: 4.0.4.1
    Win32 Version: 4.6.28619.01
    CodeBase: file:///C:/Program%20Files%20(x86)/Startup%20Manager/System.Runtime.CompilerServices.Unsafe.DLL
----------------------------------------
System.Numerics.Vectors
    Assembly Version: 4.1.4.0
    Win32 Version: 4.6.26515.06
    CodeBase: file:///C:/Program%20Files%20(x86)/Startup%20Manager/System.Numerics.Vectors.DLL
----------------------------------------

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.
```

**Comment by Arion-Kun** [comment]
Fixed in #5
